src/main.c

```diff
@@ -1043,6 +1055,54 @@ main(int argc, char** argv)
       ev_periodic_start (main_loop, &disconnect_client);
    }
 
+   if (enable_vault) {
+      /* TODO:
+       * 1. start rotate cb
+       * 2. start async vault accept cb
+       * 3. fork exec vault
+       */
+      pid_t pid;
+
+      ev_periodic_init(&rotate_frontend_password, rotate_frontend_password_cb, 0., 60, 0);
+      ev_periodic_start (main_loop, &rotate_frontend_password);
+
+      unix_vault_socket = socket(AF_INET, SOCK_STREAM, 0);
```

Comment:
Why do we need this ? The request for the current frontend password can go over the management port...